content/en/guides/deployment/nginx-proxy.md

```diff
@@ -89,6 +89,9 @@ server {
         add_header Strict-Transport-Security "max-age=31536000; includeSubDomains; preload" always;
         add_header X-Frame-Options "SAMEORIGIN";
 
+        error_page 420 = @proxy;
```

Comment:
Why 420 is being used here ?

Reply:
Any non-existent HTTP code for use inside "if". The real code will be passed from "location @proxy"
if ( $is_args ) { return 420; } => 420 code handles @proxy